src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -374,7 +387,11 @@ void EnzymesSelectorWidget::sl_saveSelectionToFile() {
     LastUsedDirHelper dir;
     dir.url = U2FileDialog::getSaveFileName(this, tr("Select enzymes selection"), dir.dir);
     if (!dir.url.isEmpty()) {
-        QFile data(dir.url);
+        QString urlToSave = dir.url;
+        if (!urlToSave.endsWith(".txt")) {
```

Comment:
You should change the file name: a user was very explicit which name he wants.

Reply:
You mean, you should not? Or what?